config/events/worker-events.yaml

```diff
@@ -28,4 +28,232 @@ events:
+    message: WorkerSliceConfig updated.
+  - name: WorkerSliceHealthUpdated
+    reason: WorkerSliceHealthUpdated
+    action: ReconcileWorkerSliceConfig
+    type: Normal
+    reportingController: worker
+    message: WorkerSliceHealth updated.
+  - name: WorkerSliceHealthUpdateFailed
+    reason: WorkerSliceHealthUpdateFailed
+    action: ReconcileWorkerSliceConfig
+    type: Warning
+    reportingController: worker
+    message: WorkerSliceHealth update failed.
+  - name: SliceGWCreated
+    reason: SliceGWCreated
+    action: CreateSliceGW
+    type: Normal
+    reportingController: worker
+    message: Slice GateWay created.
+  - name: SliceGWDeleted
+    reason: SliceGWDeleted
+    action: DeleteSliceGW
+    type: Normal
+    reportingController: worker
+    message: Slice GateWay deleted.
+  - name: SliceGWUpdated
+    reason: SliceGWUpdated
+    action: UpdateSliceGW
+    type: Normal
+    reportingController: worker
+    message: Slice GateWay updated.
+  - name: SliceGWCreateFailed
+    reason: SliceGWCreateFailed
+    action: CreateSliceGW
+    type: Warning
+    reportingController: worker
+    message: Slice GateWay creation failed.
+  - name: SliceGWDeleteFailed
+    reason: SliceGWDeleteFailed
+    action: DeleteSliceGW
+    type: Warning
+    reportingController: worker
+    message: Slice GateWay delete failed.
+  - name: SliceGWUpdateFailed
+    reason: SliceGWUpdateFailed
+    action: UpdateSliceGW
+    type: Warning
+    reportingController: worker
+    message: Slice GateWay update failed.
+  - name: SliceGWPodReconcileFailed
+    reason: SliceGWPodReconcileFailed
+    action: ReconcileSliceGWPod
+    type: Warning
+    reportingController: worker
+    message: Slice GateWay pod reconcilation failed.
+  - name: SliceGWConnectionContextFailed
+    reason: SliceGWConnectionContextFailed
+    action: ReconcileSliceGWPod
+    type: Warning
+    reportingController: worker
+    message: Slice GateWay Connection context has failed.
+  - name: SliceRouterConnectionContextFailed
+    reason: SliceRouterConnectionContextFailed
+    action: ReconcileSliceGWPod
+    type: Warning
+    reportingController: worker
+    message: Slice Router connection context has failed.
+  - name: SliceNetopQoSSyncFailed
+    reason: SliceNetopQoSSyncFailed
+    action: ReconcileSliceGWPod
+    type: Warning
+    reportingController: worker
+    message: Slice NetOp QoS sync failed.
+  - name: SliceGWRebalancingFailed
+    reason: SliceGWRebalancingFailed
+    action: ReconcileSliceGWPod
+    type: Warning
+    reportingController: worker
+    message: Slice GateWay rebalancing failed.
+  - name: SliceGWRemotePodSyncFailed
+    reason: SliceGWRemotePodSyncFailed
+    action: ReconcileSliceGWPod
+    type: Warning
+    reportingController: worker
+    message: Slice GateWay Remote pod sync failed.
+  - name: SliceGWRebalancingSuccess
+    reason: SliceGWRebalancingSuccess
+    action: ReconcileSliceGWPod
+    type: Warning
```

Comment:
Should not the type be `Normal` for a successful operation?

Reply:
Yes, you are right. Fixed it now